content/vault/v1.21.x/content/docs/enterprise/namespaces/namespace-structure.mdx

```diff
@@ -386,5 +386,4 @@ repeatable processes, and reduce human error.
 To learn more, review the following tutorials:
 
 - [Secure multi-tenancy with namespaces](/vault/tutorials/enterprise/namespaces)
-- [Vault recommended patterns](/vault/tutorials/recommended-patterns)
 - [Vault standard operating procedures](/vault/tutorials/standard-procedures)
```

Comment:
Sorry, we need to remove standard procedures as well since we don't plan on maintaining them. Let's add another tutorial which is relevant to this topic.

```suggestion
- [Manage secrets across namespaces](/vault/tutorials/enterprise/namespaces-secrets-sharing)
```